### ISSUE TITLE
F-droid MozStumbler version is still 1.7.14

### ISSUE THREAD

**Issue by jeremy447** [mozilla/MozStumbler#1795]
Is it possible to update the app in F-droid please ?

Thanks !

**Comment by crankycoder** [comment]
Waiting on the CI build server to complete - I can't seem to get Gradle 2.10 to work on my local machine with the fdroid build server. 

https://gitlab.com/fdroid/fdroiddata/merge_requests/1490/builds

**Comment by cascheberg** [comment]
Google Play Store is also not up-to-date, still at 1.8.5.

**Comment by crankycoder** [comment]
1.8.6 is merged into FDroid : https://gitlab.com/fdroid/fdroiddata/commit/5127bb6fa37e68152e0fb46c273ee536d5731f81

**Comment by jeremy447** [comment]
Thank you very much ! Is it normal that the v1.8.6 is still not available and there is some error in the build log ?

https://f-droid.org/wiki/page/org.mozilla.mozstumbler#1.8.6
https://f-droid.org/wiki/page/org.mozilla.mozstumbler/lastbuild_1080600

**Comment by jeremy447** [comment]
The builds still fails on Fdroid.

**Comment by lua86** [comment]
Try one of: Available gradle versions: 2.13 2.12 2.11 2.10 2.9 2.8 2.7 2.6 2.5 2.4 2.3 2.2.1 2.1 1.12 1.11 1.10 1.9 1.8 1.7 1.6 1.4

**Comment by jeremy447** [comment]
It seems that they use gradle 2.2 whereas it's not available.

**Comment by lua86** [comment]
In: https://gitlab.com/fdroid/fdroiddata/commit/5127bb6fa37e68152e0fb46c273ee536d5731f81
gradleVersion = "2.2"
It should be one of the availables, for example:
gradleVersion = "2.2.1"

**Comment by jeremy447** [comment]
I reopened an issue

https://github.com/mozilla/MozStumbler/issues/1797

**Comment by jeremy447** [comment]
The 1.8.6 build is available !

**Comment by chpio** [comment]
yeah, it's available but it's not marked as _latest_

**Comment by jeremy447** [comment]
Don't seem important for now since it's the latest version that is available

https://github.com/mozilla/MozStumbler/releases
